File: app/src/main/java/com/neome/api/ent/entAside/msg/MsgSpreadsheetRefFieldDataGet.java
// neome.ai API - do not change
//

package com.neome.api.ent.entAside.msg;

import com.neome.api.core.base.msg.MsgVersion;
import com.neome.api.meta.base.Types.MetaIdComp;
import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.MetaIdForm;
import com.neome.api.meta.base.Types.MetaIdSpreadsheet;
import com.neome.api.meta.base.dto.FormValueRaw;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class MsgSpreadsheetRefFieldDataGet extends MsgVersion
{
  @Nullable
  public Boolean ascOrder;

  public MetaIdForm formId;

  @Nullable
  public MetaIdSpreadsheet formSpreadsheetId;

  public FormValueRaw inputFormValueRaw;

  public MetaIdComp refFieldId;

  @Nullable
  public MetaIdField[] sortByFieldIdSet;
}

File: app/src/main/java/com/neome/api/ent/entAside/msg/MsgSpreadsheetRefFieldDataGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entAside.msg

import com.neome.api.meta.base.dto.FormValueRaw
import com.neome.api.meta.base.Types.MetaIdComp
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdSpreadsheet
import com.neome.api.core.base.msg.MsgVersion

interface MsgSpreadsheetRefFieldDataGet : MsgVersion
{
  val ascOrder: Boolean?
  val formId: MetaIdForm
  val formSpreadsheetId: MetaIdSpreadsheet?
  val inputFormValueRaw: FormValueRaw
  val refFieldId: MetaIdComp
  val sortByFieldIdSet: List<MetaIdField>?
}